broadinstitute/single_cell_portal_core
# fix gene list display when there are no clusters (SCP-3734)

SCP-3734 Previously, if a study did not have cluster data, gene lists could not be viewed and if there was also no sequence data, the explore tab itself would be unselectable.  This fixes both those cases.  This also now defaults selection to the first gene list if there is nothing else to display

Morpheus' behavior on rendering gene list heatmaps was absolutely baffling to me.  Unless there was a meaningless url param added, the gene list heatmap would look like this:
![image](https://user-images.githubusercontent.com/2800795/139106548-8471fb9a-deaa-4ba0-a3cb-cd11c9b2bdc4.png)
A quick investigation didn't yield any likely culprits for the underlying reason, and I wasn't inclined to spend a lot of time digging for a relatively rare edge case

TO TEST:
1. run `SyntheticStudyPopulator.populate('cow_liver')` to generate a study with just gene lists and nothing else
2. view the study, and confirm you can select the Explore tab, and that the gene list is automatically selected
